### PR TITLE
[YUNIKORN-1288] check for pseudo version

### PR DESCRIPTION
**Pull request by wilfred-s**
### What is this PR for?
Enforce pseudo version for the scheduler interface repo in the master.

### What type of PR is it?
* [X] - Improvement

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-1288

### How should this be tested?
The check runs as part of the PR workflow for the master

### CODE REVIEW

**Review by HuangTing-Yao — APPROVED** [review]
LGTM! Thanks @wilfred-s

**Review by craigcondit — COMMENTED** [review]
Question regarding removal of OS var.

File: Makefile [hunk 1]
```diff
@@ -110,7 +110,6 @@ check_scripts: install_shellcheck
 # from the Makefile. That caused the pull-request license check run from the github action to
 # always pass. The syntax for find is slightly different too but that at least works in a similar
 # way on both Mac and Linux. Excluding all .git* files from the checks.
-OS := $(shell uname -s | tr '[:upper:]' '[:lower:]')
```
[the remark this answers is not in the document]
Comment (craigcondit) [reply Makefile:113]: Never mind, i see there was a duplicate in the original.

**Review by craigcondit — APPROVED** [review]
+1 LGTM.

**Pull request closed without merging**